utils/tests/artifactoryconsts.go

```diff
@@ -56,6 +57,7 @@ const (
 	GoLocalRepositoryConfig                = "go_local_repository_config.json"
 	RepoDetailsUrl                         = "api/repositories/"
 	CopyItemsSpec                          = "copy_items_spec.json"
+	CopyFileWithParenthesesSpec            = "upload_file_with_parentheses.json"
```

Comment:
Should this constant name be *UploadFileWithParenthesesSpec*?

Reply:
Yes, Done.